.changeset/cute-taxis-double.md

```diff
@@ -0,0 +1,15 @@
+---
+'@backstage/backend-defaults': minor
+'@backstage/integration': minor
```

Comment:
```suggestion
'@backstage/backend-defaults': patch
'@backstage/integration': patch
```

There's no breaking changes in this, you are just adding the support for `token` as well. If this is `patch` then the Maintainers would be open to including this in a patch of the current `1.43.x` release.